backend/src/main/scala/pl/smtc/smartwords/SmartWordsApp.scala

```diff
@@ -9,17 +9,20 @@ import org.http4s.ember.server._
 import pl.smtc.smartwords.controller._
 import pl.smtc.smartwords.database._
 
+import scala.concurrent.duration.DurationInt
+
 object SmartWordsApp extends IOApp {
 
   override def run(args: List[String]): IO[ExitCode] = {
     val wordDB: WordDatabase = new WordDatabase()
     val wordController: WordController = new WordController(wordDB)
     val quizController: QuizController = new QuizController(wordDB)
 
-    if (wordDB.initDatabase()) {
+    if (wordDB.loadDatabase()) {
+      val config = CORSConfig(anyOrigin = true, allowCredentials = true, 1.day.toSeconds, anyMethod = true)
```

Comment:
Fixes related to UI communication problem (which finally was misleading by browser since the issue was in the missing array of word definitions when user used UI to add/edit word)

Reply:
Above is just a comment fo future reference 😃
I'll leave this explicit `CORSConfig` variable in case of future changes (won't have to google it)